schemas/context/person-name.schema.json

```diff
@@ -50,5 +50,5 @@
       "$ref": "#/definitions/personname"
     }
   ],
-  "meta:status": "experimental"
+  "meta:status": "stabilizing"
```

Comment:
#235 is still open, so is #227 👎

Reply:
I am good with the fixes for #235, so we can include those in the milestone.  #227 will be additive, so should not prevent us from moving this forward.  Person/PersonName is an important part of the Profile schema, so we can't really drop it from the milestone.